live/startup

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env sh
+# Start a browser to connect to Agama's web user interface skipping the authentication.
+
+TOKEN_FILE=/run/agama/token
+TOKEN=$(cat $TOKEN_FILE)
+
+firefox --kiosk --profile $HOME/.mozilla/firefox/profile "http://localhost/login?token=$TOKEN"
```

Comment:
The code is simpler without using a helper `token.html` file on the other hand the token is then visible in the `ps` output. I'm not sure how serious is this problem... :thinking:

Reply:
Yes, once you are `root` you can do everything...

The problem is that for running simple `ps` you do not need to be root. If e.g. the avahi daemon running as `avahi` user is vulnerable and allows arbitrary code execution then you could still get the root token if you force it to run `ps` and get the output although it's not running as `root`.

Reply:
OK, let's give the homepage setting a try.